### ISSUE TITLE
Consider listing issues for repos

### ISSUE THREAD

**Issue by RamblingCookieMonster** [octobox/octobox#126]
Hi!

A common issue I run into is that I lose track of issues across repos I own or help with - whether these issues are opened by me, or others, and whether they are assigned to me or not, it might be handy to see have quick access to them.

Not sure how you would target this - watching?  source repos you have access to?

No worries if this doesn't make sense here.  Can always [craft a URL like this](https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3ARamblingCookieMonster).

Cheers!

**Comment by andrew** [comment]
It's a problem I have too, we may be able to do it after #4 although not sure if makes sense within the current UI

**Comment by andrew** [comment]
I think this is out of scope for the project at the moment
